apps/cookbook/src/styles.scss

```
@@ -45,3 +45,11 @@ html {
 *:after {
   box-sizing: inherit;
 }
+
+// The <cookbook-code-viewer> component prefixes the class attribute with "language-" on <code> elements.
+// Do not target those.
+code:not([class*='language-']) {
+  border-radius: 4px;
```

Please use our design tokens instead
```suggestion
  border-radius: size(`xxxs`);
```